sandbox/basic-sqlite/index-with-types.ts

```diff
@@ -0,0 +1,48 @@
+import { Prisma, PrismaClient } from '.prisma/client'
+
+async function main() {
+  const prisma = new PrismaClient()
+
+  const email = `user.${Date.now()}@prisma.io`
+  await prisma.user.create({
+    data: {
+      email: email,
+    },
+  })
+
+  // const promiseFindManyUser = prisma.user.findMany()
```

Comment:
Still needed?

Reply:
Good point => I cleaned up the overall changes to the sandbox folders to only introduce the new sandbox for the query compiler